components/DurationComponent.tsx

```diff
@@ -27,6 +27,7 @@ export function DurationComponent(props: {
         label={"Duration"}
         type={"number"}
         id={"vsmObjectTime"}
+        min={"0"}
```

Comment:
Since the state `time` is a number I would use `min={0}` instead of the string representation.

Reply:
Will do 👍